service/src/main/java/uk/nhs/adaptors/gp2gp/common/task/EhrStatusConsumerService.java

```diff
@@ -28,6 +28,10 @@
 
     @Autowired
     private EhrExtractStatusRepository ehrExtractStatusRepository;
+    @Autowired
+    private IdGenerationService idGenerationService;
+    @Autowired
+    private XPathService xPathService;
```

Comment:
I think it was ok to have that static

Reply:
With autowired XPath object it needs to be a bean.

Reply:
So don't use `XPath` as a dependency as per my other comment